LICENCE.rst

```diff
@@ -1,8 +1,10 @@
 GrantNav
 ========
 
+Note: This license doesn't apply to the 360Giving theme and related assets (such as the logo).
```

Comment:
My feeling is that 'related' leaves too much to interpretation, especially as this file doesn't describe the license that is attached to those works. 

Are there any works in this repo that aren't covered by this license? If there aren't, that could be a neat way of distinguishing.